pkg/src/message.js

```diff
@@ -8,8 +8,14 @@ import {
 /**
  * @param {import('../index.d.ts').Message} m
  * @param {import('./utils.js').Pkg} pkg
+ * @param {import('../utils.d.ts').FormatMessageOpt} opt
  */
-export function formatMessage(m, pkg) {
+export function formatMessage(m, pkg, opt = {}) {
+  // TODO: defaults to false instead of true
+  const color = opt.color ?? true;
+  const bold = color ? c.bold : (s) => s
+  const yellow = color ? c.yellow : (s) => s
```

Comment:
`picocolors` exports `createColor(enabled: boolean)` which I think we can use here instead. That way we don't have to create the variable here and rename all `c.bold` to `bold`. I think we can cache and hoist the returned result of `createColor` up as a variable outside of this function.

Also, I'm thinking maybe we can have `opts.color: boolean | undefined` so that:
- `true`: Force has color
- `false`: Force no color
- `undefined`: Default to whether the environment supports color (already [handled by `picocolors`](https://github.com/alexeyraspopov/picocolors/blob/6f6011182c0f16cf119019ca9bef1fadbe86b913/picocolors.js#L3-L9) by default)